src/insights.jsx

```diff
@@ -65,7 +65,8 @@ function spawn_error_to_string(err, data) {
     }
     // When err contains only string then return this string
     if (err) {
-        return err;
+        // force the conversion of the error to string
+        return err + "";
```

Comment:
Shouldn't we use `err.toString()` instead? More details here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString

Reply:
This is all redundant. cockpit has the cockpit.message function which returns arg.message  if it's present else it parses the problem for known failures. https://github.com/cockpit-project/cockpit/blob/main/pkg/lib/cockpit.js#L3941

Reply:
In other words please just remove this whole function and start using cockpit.message instead.

Reply:
What I needed was to show the whole program stdout+stderr as error message, and in my past testing `cockpit.message()` did not seem to work for that; see https://github.com/candlepin/subscription-manager-cockpit/pull/20

Of course, I could have screwed up something while testing back then...

Reply:
Discussed over on irc, land this fix as is with the toString change, and we need to improve the error reports in general separately.